### PR TITLE
Randy searches

### PR DESCRIPTION
**Pull request by Rando-cal**
Added searchbars for Admin:Users, Admin:Homes, Admin:Surveys.

### CODE REVIEW

File: frontend/front/src/pages/Admin/user/UserTable.js
Comment (mattdelsordo) [inline frontend/front/src/pages/Admin/user/UserTable.js:1]: This all looks good, but it looks like changing this table has broken the link to the individual user page when clicking on a row.

**Review by mattdelsordo — CHANGES_REQUESTED** [review]

File: frontend/front/src/pages/Admin/user/UserTable.js
Comment (Rando-cal) [reply frontend/front/src/pages/Admin/user/UserTable.js:1]: @mattdelsordo User row click issue fixed. Now it should direct to the actual user. If your are not seeing the Search bar in the table, then make sure to do "yarn add @mui/x-data-grid"

**Comment by Rando-cal** [comment]
Want to meet with someone else to resolve conflicts...

**Comment by Rando-cal** [comment]
@mattdelsordo @LWest001 @EfrainAD 

Can someone review this to get it through?  Thanks!!

**Comment by mattdelsordo** [comment]
> @mattdelsordo @LWest001 @EfrainAD
> 
> Can someone review this to get it through? Thanks!!

Looks good to me but needs a rebase + merge conflict resolution

**Review by mattdelsordo — APPROVED** [review]

**Comment by Rando-cal** [comment]
I think this is moot

**Pull request closed without merging**